modules/schematics/src/entity/files/__name@dasherize@if-flat__/__name@dasherize@group-reducers__.reducer.ts.template

```diff
@@ -15,39 +13,47 @@ export const initialState: State = adapter.getInitialState({
   // additional entity state properties
 });
 
-export const reducer = createReducer(
-  initialState,
-  on(<%= classify(name) %>Actions.add<%= classify(name) %>,
-    (state, action) => adapter.addOne(action.<%= camelize(name) %>, state)
-  ),
-  on(<%= classify(name) %>Actions.upsert<%= classify(name) %>,
-    (state, action) => adapter.upsertOne(action.<%= camelize(name) %>, state)
-  ),
-  on(<%= classify(name) %>Actions.add<%= classify(name) %>s,
-    (state, action) => adapter.addMany(action.<%= camelize(name) %>s, state)
-  ),
-  on(<%= classify(name) %>Actions.upsert<%= classify(name) %>s,
-    (state, action) => adapter.upsertMany(action.<%= camelize(name) %>s, state)
-  ),
-  on(<%= classify(name) %>Actions.update<%= classify(name) %>,
-    (state, action) => adapter.updateOne(action.<%= camelize(name) %>, state)
-  ),
-  on(<%= classify(name) %>Actions.update<%= classify(name) %>s,
-    (state, action) => adapter.updateMany(action.<%= camelize(name) %>s, state)
-  ),
-  on(<%= classify(name) %>Actions.delete<%= classify(name) %>,
-    (state, action) => adapter.removeOne(action.id, state)
-  ),
-  on(<%= classify(name) %>Actions.delete<%= classify(name) %>s,
-    (state, action) => adapter.removeMany(action.ids, state)
-  ),
-  on(<%= classify(name) %>Actions.load<%= classify(name) %>s,
-    (state, action) => adapter.setAll(action.<%= camelize(name) %>s, state)
-  ),
-  on(<%= classify(name) %>Actions.clear<%= classify(name) %>s,
-    state => adapter.removeAll(state)
-  ),
-);
+const feature = createFeature({
```

Comment:
the feature can also be exported

```suggestion
export const feature = createFeature({
```

Reply:
thoughts on `export const <%= pluralize(name) %>Feature = ...`? 👀